priv/perf/apps/load_test/lib/child_chain/exit.ex

```diff
@@ -84,6 +84,7 @@ defmodule LoadTest.ChildChain.Exit do
 
     {:ok, receipt_enc} =
       Ethereumex.HttpClient.eth_call(%{
+        from: Application.fetch_env!(:load_test, :eth_call_from_address),
```

Comment:
why is this a special case? why not use authority address, or plasma framework contract address?

Reply:
This is not under umbrella app. We do not have authority address here. We could use contract address to make it simple. That's what I'll do.